tools/msftidy.rb

```diff
@@ -601,6 +601,18 @@ def check_invalid_url_scheme
     end
   end
 
+  # This checks looks for extranous commas
+  # Stuff with comments should match too: , #metasploit module ]
+  # but it should not match interpolation #{
+  def check_comma
+    test = @source.scan(/[^\n]+,(?:\s*#[^{][^\n]+)?\s*[\]\}]/)
+    unless test.empty?
+      test.each { |item|
```

Comment:
Maybe make the same change on [line 598](https://github.com/rapid7/metasploit-framework/pull/4593/files#diff-ff5f6b3f4c780f2d2beb8c10726223b6R598).